packages/wallet-sdk/src/provider/MobileRelayUI.ts

```diff
@@ -0,0 +1,125 @@
+import { ErrorHandler } from "../errors";
+import {
+  EthereumAddressFromSignedMessageRequest,
+  SignEthereumMessageRequest,
+  SignEthereumTransactionRequest,
+  SubmitEthereumTransactionRequest,
+} from "../relay/Web3Request";
+import {
+  EthereumAddressFromSignedMessageResponse,
+  SignEthereumMessageResponse,
+  SignEthereumTransactionResponse,
+  SubmitEthereumTransactionResponse,
+} from "../relay/Web3Response";
+import { AddressString, ProviderType } from "../types";
+import { WalletUI, WalletUIOptions } from "./WalletUI";
+
+// TODO: Implement & present in-page wallet picker instead of navigating to www.coinbase.com/connect-dapp
+export class MobileRelayUI implements WalletUI {
```

Comment:
empty class conforming to `WalletUI`, which skips the creation of QR code dialogs and avoids running any business logic associated with desktop connection cases.